### ISSUE TITLE
filter "Switch files" list

### ISSUE THREAD

**Issue by frederich** [slynch8/10x#1308]
It's not possible to filter "switch files". You one switch file and don't do a hole workspace search, for instance in the workspace are 40 files with the name "main.cpp" and you has still open the right one, but your "switch file" list has 50 entries, then I just want to filter "switch files" list.

**Comment by slynch8** [comment]
do you mean the ctrl+tab switch file panel? I'm not sure I understand, that should only show the currently open files

**Comment by frederich** [comment]
Yes, there is a different behaviour of the "switch files" panel. Via short-cut it opens, you select and it closes. Via menu it stays open as long as you want. 

When I have many files open, I need an easy way to switch between them. In Emacs you can do this with a search filter in the open files (Emacs buffers). A good solution would also be that you can filter in workspace search only in the filenames of the currently open files. Or that the switch-files panel stays open and you can filter in it.
